src/App.jsx

```diff
@@ -10,8 +10,46 @@ function isAffectedBrowser() {
   const ios = navigator.userAgent.includes('iPhone OS 15')
   const ipad = navigator.userAgent.includes('iPad; CPU OS 15')
   const macos = navigator.userAgent.includes('Macintosh') && navigator.userAgent.includes('Version/15')
+  const features = detectBasedOnFeatures()
 
-  return ios || ipad || macos
+  return ios || ipad || macos || features
```

Comment:
Since the code still relies on the UA, there may be false positives. But they should be very unlikely.